### PR TITLE
Enh/extract method transformation

### PR DESCRIPTION
**Pull request by Ducasse**
Cleaning some aspects of extract method transformation

### CODE REVIEW

File: src/Refactoring2-Transformations/RBExtractMethodTransformation.class.st [hunk 1]
```diff
@@ -252,6 +260,13 @@ RBExtractMethodTransformation >> generateNewMethodWith: aMethodName [
 	^ newMethodNode
 ]
 
+{ #category : #api }
```
Comment (jecisc) [inline src/Refactoring2-Transformations/RBExtractMethodTransformation.class.st]:
```suggestion
{ #category : #initialization }
```

The protocol is wrong here

File: src/SystemCommands-SourceCodeCommands/SycExtractMethod2Command.class.st [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+	invalidArgNames := Set new.
+	(model allImplementorsOf: aSymbol) do: [ :e |
+		| mth |
+		mth := e realClass >> aSymbol.
+		invalidArgNames
+			addAll: mth argumentNames;
+			addAll: mth origin instVarNames ].
+	^ invalidArgNames
+]
+
```
Comment (jecisc) [inline src/SystemCommands-SourceCodeCommands/SycExtractMethod2Command.class.st]:
```suggestion
```

This method is already present in the trait

File: src/SystemCommands-SourceCodeCommands/SycExtractMethod2Command.class.st [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+	dialog cancelled ifTrue: [ CmdCommandAborted signal ].
+	^ methodName
+]
+
+{ #category : #initialization }
+SycExtractMethod2Command >> initializeDefaultOptionsOf: refactoring [
+
+	NautilusRefactoring new refactoringOptions: refactoring
+]
+
```
Comment (jecisc) [inline src/SystemCommands-SourceCodeCommands/SycExtractMethod2Command.class.st]:
```suggestion

```

Duplicated from trait

File: src/SystemCommands-SourceCodeCommands/SycExtractMethod2Command.class.st [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+{ #category : #initialization }
+SycExtractMethod2Command >> setUpModelFromContext: aContext [
+
+	| browser scope |
+	browser := aContext browser.
+	scope := browser defaultNavigationScope.
+	scope isCurrentImage ifTrue: [
+		scope := RefactoringSettings scopeRefactoring ].
+	model := RBClassModelFactory rbNamespace onEnvironment: scope asRBEnvironment
+]
```
Comment (jecisc) [inline src/SystemCommands-SourceCodeCommands/SycExtractMethod2Command.class.st]:
```suggestion
```

Duplicated from trait

**Review by jecisc — COMMENTED** [review]
Some release tests are failing and I commented suggestions to fix them

**Comment by Ducasse** [comment]
So I could fix them.

**Review by jecisc — APPROVED** [review]

**Pull request merged**